report/src/main/webapp/app/layout/main/url/reports/source-diff.html

```diff
@@ -19,11 +19,15 @@
 -->
 <div class="clearfix"></div>
 <div class="col-xs-6">
+  <p class="layout-compare-description text-center">Pattern
+    <span class="label label-info">{{case.comparator.stepResult.data.patternTimestamp | date:'yyyy-MM-dd HH:mm'}}</span><br>
+    <span class="label-spacer"></span>
+  </p>
```

Comment:
See the comment for `SourceComparator`. The timestamp should ideally be displayed only when it's available and truthy.

Reply:
after update it is shown only when both dates are available - otherwise date string is empty - hence it is not showed